### PR TITLE
chore: Fix broken link for Sway types

### PR DESCRIPTION
**Pull request by MartinKong1990**
Old: https://fuellabs.github.io/sway/latest/basics/index.html

New: https://fuellabs.github.io/sway/v0.47.0/book/basics/built_in_types.html

### CODE REVIEW

File: packages/abi-typegen/README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ The table below describes how Sway types are converted from/to Typescript.
 |         **options** | Option<`MyType`>               |          `Option<MyType>`           |          `Option<MyType>`           |
 | **raw untyped ptr** | `123`                          |           `BigNumberish`            |                `BN`                 |
 
-> For more info on Sway types, click [here](https://fuellabs.github.io/sway/latest/basics/index.html)
+> For more info on Sway types, click [here](https://fuellabs.github.io/sway/v0.47.0/book/basics/built_in_types.html)
```
Comment (arboleya) [inline packages/abi-typegen/README.md:113]:
```suggestion
> For more info on Sway types, click [here](https://docs.fuel.network/docs/sway/basics/built_in_types/)
```

**Review by arboleya — CHANGES_REQUESTED** [review]
Thank you.

**Comment by arboleya** [comment]
@MartinKong1990 FYI — in case you didn't see the edit on my [last comment](https://github.com/FuelLabs/fuels-ts/pull/1423#pullrequestreview-1722601846).

**Comment by MartinKong1990** [comment]
> @MartinKong1990 FYI — in case you didn't see the edit on my [last comment](https://github.com/FuelLabs/fuels-ts/pull/1423#pullrequestreview-1722601846).

Looks like need to install some software in my laptop for pnpm to generate an empty changeset. Any alternative solution?

**Comment by arboleya** [comment]
What do you mean by "additional software"?

It should be as easy as:
 
```bash
cd fuels-ts
pnpm install
pnpm changeset add --empty
```

Then `add`, `commit` and `push` the generated file.

Voilà.

**Comment by MartinKong1990** [comment]
Does resubmit pull request from different branch will solve the changeset problem?

**Comment by danielbate** [comment]
@MartinKong1990 No, the changeset is an additional file that needs to be added to all PRs, generated using the commands that @arboleya has referenced above. For this PR, we would be expecting an empty changeset, [such as in this example](https://github.com/FuelLabs/fuels-ts/pull/1422/files#diff-4780fec66cc2bb30fc9750aa35df80a80c80ae4003ca13a9d33f2062e1ef0597).

**Comment by MartinKong1990** [comment]
> @MartinKong1990 No, the changeset is an additional file that needs to be added to all PRs, generated using the commands that @arboleya has referenced above. For this PR, we would be expecting an empty changeset, [such as in this example](https://github.com/FuelLabs/fuels-ts/pull/1422/files#diff-4780fec66cc2bb30fc9750aa35df80a80c80ae4003ca13a9d33f2062e1ef0597).

@danielbate @arboleya 

Actually I use github.com web site with Windows/Chrome. No luck trying with Windows 10 and ubuntu after install different software. I suggest two possible solutions:

Solution 1: In github.com web site, I create an empty file that same as [.changeset/proud-ducks-travel.md](https://github.com/FuelLabs/fuels-ts/pull/1422/files#diff-4780fec66cc2bb30fc9750aa35df80a80c80ae4003ca13a9d33f2062e1ef0597).  with different name at my fork and push here. 

Solution 2: Close this PR. You guys submit the PR and add me as a co-author.

**Comment by arboleya** [comment]
Option 1.

**Review by Dhaiwat10 — APPROVED** [review]

**Review by arboleya — APPROVED** [review]
Thank you.

**Pull request merged**